plan.tex

```diff
@@ -57,15 +70,33 @@ \subsubsection{Middleware}
 we will have to do on IDF.
 
 \paragraph{Workflow} needs to be functioning at scale for DP0.2, ideally we should basic workflow
-early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03)
+early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03).
+
+From the construction side we have BPS as a deliverable which may be useful on IDF also.
+We shall evaluate BPS as an option later in 2020 (DP-MW-M07).
+See \citeds{LDM-636}, \citeds{LDM-633}, \citeds{DMTN-123}.
+BPS translates the quantum graph to DAGMan for execution on HTCondor and submits the jobs.
```

Comment:
Did we give up using CWL or other standard workflow format?

Reply:
no I can add CWL